notifications/rest/notifications/send-notification-detailed/send-notification-detailed.3.x.js

```diff
@@ -11,23 +11,11 @@ var service = client.notify.services('ISxxx');
 
 service.notifications
   .create({
-    "identity": "00000001",
-    "title": "Generic loooooooong title for all Bindings",
-    "body": "This is the body for all Bindings",
-    "fcm": {
-      "notification": {
-        "title": "New alert",
-        "body": "Hello Bob!"
-      }
-    },
-    "apn": {
-      "aps": {
-        "alert": {
-          "title": "New alert",
-          "body": "Hello Bob!"
-        }
-      }
-    },
-  }).then(function(notification) {
+    identity: '00000001',
+    title: 'Generic loooooooong title for all Bindings',
+    body: 'This is the body for all Bindings',
+    fcm: '{"notification":{"title":"New alert","body":"Hello Bob!"}}',
+    apn: '{"aps":{"alert":{"title":"New alert.","body":"Hello Bob!"}}}'
```

Comment:
This seems like a problem with the library. I have already notified the Yoyodine team but for now I think is better to leave it like this as it works. Wdyt?
Once the SDK is fixed I'll put back this snippet as it was before.